examples/vrptw.ipynb

```diff
@@ -266,149 +270,15 @@
    ]
   },
   {
+   "attachments": {},
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Inspect routes\n",
```

Comment:
I'm fine with replacing the diagnostics module, but can we somehow still print statistics per route? I think that is quite useful?

Reply:
I forgot to replace this bit with something new again. I'm going to do that now.

Reply:
I've added such a table back into the notebook in my most recent commit.